dashboard/templates/base.html

```diff
@@ -57,7 +57,13 @@
     {% endblock %}
     {% csrf_token %}
     <footer>
-        <div style="float: left; width: 50%">Copyright © 2018 The Regents of the University of Michigan</div>
+        {% load flatpages %}
+        {% get_flatpages as flatpages %}
```

Comment:
It looks like this could be a bit simpler if you did this
```
{% get_flatpages '/copyright/' as pages %}
<div style="float: left; width: 50%">{{ pages.first.content|safe }}</div>

```
ref: https://stackoverflow.com/questions/33767836/using-a-specific-flatpage-in-a-template

Reply:
Done